### PR TITLE
Comment embed

### PR DESCRIPTION
**Pull request by annatraussnig**

### CODE REVIEW

File: components/editor/modules/embed/index.js [hunk 1]
```diff
@@ -76,6 +77,17 @@ export const getTwitterEmbed = gql`
...
+  query getCommentEmbed($id: ID!) {
+    comments(focusId: $id) {
+      focus {
+        id
+        text
+      }
+    }
+  }
+`
+
```
Comment (annatraussnig) [inline components/editor/modules/embed/index.js:90]: @patrickvenetz the other embed queries have a matching `embed` gql queries, whereas here I used the `comments` query. Please advise.

File: components/editor/modules/embed/index.js [hunk 2]
```diff
@@ -186,6 +198,10 @@ const YOUTUBE_REGEX = /^http(?:s?):\/\/(?:www\.)?youtu(?:be\.com\/watch\?v=|\.be
 // One capturing group at match[1] that catches the video id
 const VIMEO_REGEX = /^(?:http|https)?:\/\/(?:www\.)?vimeo.com\/(?:channels\/(?:\w+\/)?|groups\/(?:[^/]*)\/videos\/|)(\d+)(?:|\/\?)$/
 
+const COMMENT_REGEX = new RegExp(
+  `^${FRONTEND_BASE_URL}/(?:(?:dialog\\?t=article&id=[a-f\\d-]+&)|(?:beitrag/diskussion\\?))focus=([a-f\\d-]+)$`
```
Comment (patrickvenetz) [inline components/editor/modules/embed/index.js:202]: Expression seems not to cover comments in «Allgemeines Feedback» come with `t=general`, like so:
- https://www.republik.ch/dialog?t=general&focus=b18a4684-2b4d-4df6-bab2-72905d06dfce

Comment (annatraussnig) [reply components/editor/modules/embed/index.js:202]: It doesn't but it should. I'll fix.

File: components/editor/modules/embed/index.js (hunk 1 above)
Comment (patrickvenetz) [reply components/editor/modules/embed/index.js:90]: Not sure about implications just yet.

But gutfeeling is to stick to your approach with `comments` API.

File: components/editor/modules/embed/index.js (hunk 2 above)
Comment (patrickvenetz) [reply components/editor/modules/embed/index.js:202]: Dedicated discussion articles, like `/something-something/diskussion`, seem not to be matched:
- https://www.republik.ch/2021/04/26/wie-geht-es-ihnen-im-corona-limbo/diskussion?focus=96e268ba-8e35-4ac0-a1e0-73766eaacbe8

File: components/Templates/withFrontData.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { graphql } from 'react-apollo'
 import gql from 'graphql-tag'
-import { TeaserActiveDebates } from '@project-r/styleguide/lib/components/TeaserActiveDebates'
-import { TeaserMyMagazine } from '@project-r/styleguide/lib/components/TeaserMyMagazine'
+import { TeaserActiveDebates } from '@project-r/styleguide/lib/components/TeaserActiveDebates' // should this be such?
+import { TeaserMyMagazine } from '@project-r/styleguide/lib/components/TeaserMyMagazine' // ditto
```
Comment (tpreusse) [inline components/Templates/withFrontData.js:4]: for the time being let's leave it as is. eventually we can properly export it, styleguide needs a setup/tooling modernisation soon

```suggestion
import { TeaserActiveDebates } from '@project-r/styleguide/lib/components/TeaserActiveDebates'
import { TeaserMyMagazine } from '@project-r/styleguide/lib/components/TeaserMyMagazine'
```

File: lib/translations.json [hunk 4]
```diff
@@ -998,10 +998,6 @@
       "key": "publish/preview/desktop",
       "value": "Desktop"
     },
-    {
-      "key": "publish/preview/404",
-      "value": "Dokument oder Schema nicht verfügbar."
-    },
```
Comment (tpreusse) [inline lib/translations.json:1004]: this was missing in the google sheet, I've added it, please rerun `npm run translations`

**Review by tpreusse — APPROVED** [review]
LGTM, two small clean ups needed and then the released styleguide bump

**Pull request merged**